spec/index.bs

```diff
@@ -543,6 +551,28 @@ The system's GPU driver may also have its own cache of compiled shaders and pipe
 may want to disable these when at all possible, or add per-partition data to shaders in ways that
 will make the GPU driver consider them different.
 
+### Driver bugs ### {#privacy-driver-bugs}
+
+In addition to the concerns outlined in [[#security-driver-bugs|Security Considerations]], driver
+bugs may introduce differences in behavior that can be observed as a method of differentiating
+users. The mitigations mentioned in Security Considerations apply here as well, including
+coordinating with GPU vendors and implementing workarounds for known issues in the user agent.
+
+### Adapter Identifiers ### {#privacy-adapter-identifiers}
+
+<div class=note>
+Note: It is expected that WebGPU will expose some level of information identifying the type of GPU
+Adapter in use. This is a potential source of fingerprinting information but past experience with
+WebGL has demonstrated that it is necessary to some degree in order to enable developers to create
+robust applicaitons and respond effectively to user issues.
+<br/><br/>
+The design of this aspect of the API is ongoing, with a primary goal being to deliver much more
```

Comment:
I don't think this paragraph needs to be in the spec.
You could add an `Issue: describe the considerations for exposing adapter info` or something like that instead

Reply:
Done!